README.md

```diff
@@ -256,6 +256,49 @@ class MyComponent
 end
 ```
 
+### `init_label_set`
+
+The time series of a metric are not initialiazed until something happens. For counters, for example, this means that the time series do not exist until the counter is increamented for the first time.
+
+This can lead to cases that are hard to handle:
+
+```ruby
+counter =
+  Counter.new(
+    :http_requests_total,
+    docstring: '...',
+    labels: [:app, :env, :version],
+  )
+
+# After declaring the counter nothing is exported because the time series have not been initialized
+
+# We call increment for the first time and the time serie for the label set provided is initialiazed
+counter.increment(labels: {app: "frontend", env: "production", version: "v1"})
+
+# Client starts exporting
+# http_requests_total{app="frontend", env="production", version="v1"} 1
+
+# We call increment again
+counter.increment(labels: {app: "frontend", env: "production", version: "v1"})
+
+# Client exports
+# http_requests_total{app="frontend", env="production", version="v1"} 2
```

Comment:
I would make this shorter and with no code. I think it's easier to read if you explain the idea that you initialize a counter, it doesn't get exported until it's incremented at least once. The key sentence to explain the problem is the one below. (And i'd make it "not code") :)

Reply:
Yeah, the code might be a little bit verbose. I'll remove it.